spec/support/ansible_shared/provider.rb

```diff
@@ -22,10 +22,12 @@
   describe "#destroy" do
     it "will remove all child objects" do
       subject.automation_manager.configured_systems = [
-        FactoryGirl.create(:configured_system, :computer_system =>
-          FactoryGirl.create(:computer_system,
+        FactoryGirl.create(:configured_system_automation_manager,
```

Comment:
What is this factory?  We don't have a ConfiguredSystemAutomationManager model or table.

Reply:
I also don't see it in the factories in manageiq

Reply:
This PR depends on https://github.com/ManageIQ/manageiq/pull/16755 which (other than the `destroy` method rewrite) has created the factory `:configured_system_automation_manager` in [here](https://github.com/ManageIQ/manageiq/pull/16755/files#diff-3e18369d4c9d44e9491f3507daaa2b08R6)